tools/compare_families.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+
+"""
+compare_families.py - Compare two families of one benchmark or it's result
+                      and report the difference.
+"""
+
+import argparse
+from argparse import ArgumentParser
+import sys
+import gbench
+from gbench import util, report
+from gbench.util import *
+
+
+def check_input(inp, flags):
+    """
+    Perform checking on the user provided input and diagnose any abnormalities
+    """
+    in_kind, in_err = classify_input_file(inp)
+    output_type = find_benchmark_flag('--benchmark_out_format=', flags)
+    if in_kind == IT_JSON and len(flags) > 0:
+        print("WARNING: passing --benchmark flags has no effect since the "
+              "input is JSON")
+    if output_type is not None and output_type != 'json':
+        print(("ERROR: passing '--benchmark_out_format=%s' to "
+               "'compare_families.py` is not supported.") % output_type)
+        sys.exit(1)
+
+
+def main():
+    parser = ArgumentParser(
+        description='compare two families of one benchmark')
+    parser.add_argument(
+        'test',
+        metavar='test',
+        type=str,
+        nargs=1,
+        help='A benchmark executable or JSON output file')
+    parser.add_argument(
+        'family1',
```

Comment:
maybe name this 'baseline_family' or something?

Reply:
Yes, probably. Will change.